pkgs/tools/misc/wayback/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, google-chrome
+}:
+
+buildGoModule rec {
+  pname = "wayback";
+  version = "0.19.1";
+
+  src = fetchFromGitHub {
+    owner = "wabarc";
+    repo = "wayback";
+    rev = "v${version}";
+    hash = "sha256-LIWCT0/5T52VQQK4Dy6EFmFlJ02MkfvKddN/O/5zpZc=";
+  };
+
+  vendorHash = "sha256-TC4uwJswpD5oKqF/rpXqU/h+k0jErwhguT/LkdBA83Y=";
+
+  doCheck = false;
+
+  buildInputs = [
+    google-chrome
```

Comment:
Why did you switch to chrome? That has an unfree license and now ofborg cannot build the package.

Reply:
I was testing build on linux and forgot to switch it back. Sorry for bothering you.

Reply:
It doesn't really bother me, it just delays the merge of this PR.